### PR TITLE
Merge r1.4 bugfixes to main

### PR DESCRIPTION
**Pull request by ericharper**

### CODE REVIEW

File: requirements/requirements_lightning.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-pytorch-lightning>=1.4.0
+pytorch-lightning>=1.4.8,<1.5
```
Comment (blisc) [inline requirements/requirements_lightning.txt:1]: Let's remove the `<1.5` for main

**Review by blisc — COMMENTED** [review]
TTS looks good to go

File: requirements/requirements_lightning.txt (hunk 1 above)
Comment (ericharper) [reply requirements/requirements_lightning.txt:1]: We know that 1.5 has breaking API changes. We should probably relax that after we merge those fixes into main?

**Comment by aklife97** [comment]
Transformer Decoder changes look good!

File: requirements/requirements_lightning.txt (hunk 1 above)
Comment (blisc) [reply requirements/requirements_lightning.txt:1]: But how will I know when lightning gets updated to 1.5 if there are no complaints of bugs in our slack channel?

**Review by titu1994 — APPROVED** [review]
LGTM

**Comment by ericharper** [comment]
Removing upper bound on PTL after api fixes are in main.

**Pull request merged**